server/graphql/common/collective.js

```diff
@@ -16,4 +26,65 @@ async function hostResolver(collective, _, { loaders }) {
   return hostCollective;
 }
 
-export { hostResolver };
+async function sendMessage({ req, collective, args, isGqlV2 }) {
+  const user = req.remoteUser;
```

Comment:
To check for OAuth scope

```suggestion
  checkRemoteUserCanUseAccount(req);
  const user = req.remoteUser;
```